### ISSUE TITLE
[Tracking]  Star Citizen (RSI) caching not possible due to ssl

### ISSUE THREAD

**Issue by Lepidopterist** [uklans/cache-domains#33]
This is a tracking issue to collate any progress with caching Star Citizen

Currently it is not possible to cache Star Citizen using steamcache/generic due to its reliance on SSL

**Comment by attilatdeak** [comment]
Isnt the SC launcher using torrent to distribute the update files? So, essentially you could grab the torrents and add a local seeder, so when they connect to the updater they would esentially connect to the cache instance as a leecher and download from the cache on local network speeds?

**Comment by GotenXiao** [comment]
No, the RSI launcher downloads content over HTTPS - it hasn't used torrents in years. This can be trivially verified on Windows through Resource Monitor's Network tab. At time of writing, manifests and content are typically downloaded from a Google CDN endpoint.
